app/views/issues/_sidebar.html.erb

```diff
@@ -9,12 +9,12 @@
 
 <%= render 'shared/sidebar_collection',
   category_id: nil,
-  collection: @issues,
+  collection: @issues.sort,
```

Comment:
Is this an intended behaviour change?

Reply:
Without this, I noticed the issues in the sidebar were not rendering in the same order as `issues#show`.